test/TritonGPU/amd/amd-reorder-instructions.mlir

```diff
@@ -23,3 +23,2317 @@ module attributes {"triton_gpu.num-warps" = 8 : i32, "triton_gpu.threads-per-war
     tt.return
   }
 }
+
+// -----
+#blocked = #triton_gpu.blocked<{sizePerThread = [1, 4], threadsPerWarp = [1, 32], warpsPerCTA = [4, 1], order = [1, 0]}>
```

Comment:
did you mean to add this in this PR? This seems unrelated?

Reply:
These tests ensure that after pipelining the loop the reorder-instructions pass will move tt.load's as early as possible. This increases register pressure but enables the overlap of the global loads with the compute of the previous tile.

I will add a description in the test of what is expected.